frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss

```diff
@@ -80,14 +80,14 @@
             --lemon-button-font-size: 0.75rem;
             --lemon-button-height: 1.6875rem;
             --lemon-button-gap: 0.25rem;
-            --lemon-button-side-action-icon-width: 1.25rem;
+            --lemon-button-side-action-width: 1.5rem;
```

Comment:
Should this value be computed or configurable?

In the case of the small & default options it seems the new `--lemon-button-side-action-width` equals the old `--lemon-button-side-action-icon-width` + `--lemon-button-padding-horizontal`. 

In this case there is a drift (`0.375rem + 1.25rem =/= 1.5rem`). I'm surprised no snapshots changed but my guess is that we don't snapshot the `xsmall` button with a side action